src/main/scala/cognite/spark/v1/EventsRelation.scala

```diff
@@ -133,16 +132,16 @@ object EventsRelation extends UpsertSchema {
 
 final case class EventsUpsertSchema(
     id: Option[Long] = None,
-    startTime: Option[Instant] = None,
-    endTime: Option[Instant] = None,
-    description: Option[String] = None,
-    `type`: Option[String] = None,
-    subtype: Option[String] = None,
+    startTime: OptionalField[Instant] = FieldNotSpecified,
+    endTime: OptionalField[Instant] = FieldNotSpecified,
+    description: OptionalField[String] = FieldNotSpecified,
+    `type`: OptionalField[String] = FieldNotSpecified,
+    subtype: OptionalField[String] = FieldNotSpecified,
     metadata: Option[Map[String, String]] = None,
     assetIds: Option[Seq[Long]] = None,
```

Comment:
Are these not nullable?

Reply:
these two are of type `NonNullableSetter`, they can not be updated to be null. You will want to update them with an empty collection to clear it.

Reply:
I get it thanks 🚀